### ISSUE TITLE
Wrong symbol on name applied with extension method

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#905]
```scala
/* ONLY
rewrite = DelayedInitToMain
*/
package test

object DelayedInitToMain extends App {
  println(args.toList)
}
```
Produces the following symbol for `args`

```
[102..106): args => _root_.scala.Predef.refArrayOps([Ljava/lang/Object;)[Ljava/lang/Object;.
```

I expected the symbol to be 
```scala
[102..106): args => _root_.scala.App#args()[Ljava/lang/String;.
```

**Comment by xeno-by** [comment]
Yeah, `refArrayOps` should be in sugars.

**Comment by olafurpg** [comment]
Pending fix in #988

**Comment by olafurpg** [comment]
Fixed in #988. The names section for that example is now

```
[info]   ===========
[info]   => Obtained
[info]   ===========
[info]   [9..13): test => _root_.test.
[info]   [22..39): DelayedInitToMain => _root_.test.DelayedInitToMain.
[info]   [48..51): App => _root_.scala.App#
[info]   [52..52):  => _root_.java.lang.Object#`<init>`()V.
[info]   [56..63): println => _root_.scala.Predef.println(Ljava/lang/Object;)V.
[info]   [64..68): args => _root_.scala.App#args()[Ljava/lang/String;.
[info]   [69..75): toList => _root_.scala.collection.TraversableOnce#toList()Lscala/collection/immutable/List;.
```
